WebRoot/js/zimbraMail/abook/view/ZmContactPicker.js

```diff
@@ -420,12 +420,13 @@ function() {
             this._searchInSelect.addOption(ZmMsg.GAL, true, ZmContactsApp.SEARCHFOR_GAL);
         }
 
+		this._searchInSelect.addOption("Organizational Address Book", false, 4);
```

Comment:
need to add same ui option in case of multiple accounts also, check if condition above this block

Reply:
Fixed.